nemo_text_processing/text_normalization/data_loader_utils.py

```diff
@@ -140,9 +140,9 @@ def evaluate(preds: List[str], labels: List[str], input: Optional[List[str]] = N
             acc = acc + 1
         else:
             if input:
-                print(f"inpu: {json.dumps(input[i])}")
-            print(f"gold: {json.dumps(label_norm)}")
-            print(f"pred: {json.dumps(pred_norm)}")
+                print(f"input: {json.dumps(input[i], ensure_ascii=False)}")
```

Comment:
use ensure_ascii = true. it's safer for text processing. this will destroy an entire pipeline.